LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/ADR/LoRaADRManagerBase.cs

```diff
@@ -32,16 +32,15 @@ public virtual async Task StoreADREntryAsync(LoRaADRTableEntry newEntry)
                 return;
             }
 
-            if (string.IsNullOrEmpty(newEntry.DevEUI) ||
-               string.IsNullOrEmpty(newEntry.GatewayId))
+            if (string.IsNullOrEmpty(newEntry.GatewayId))
```

Comment:
To be consistent with the previous semantics this should add back the check `!newEntry.DevEUI.IsValid` and the message should read `"Missing Gateway ID or invalid DevEUI"`.

Reply:
Done in 0f086f54ffac5a62ee2933d0ef5368314920c7ef